src/pages/workspace/accounting/xero/export/XeroPurchaseBillStatusSelectorPage.tsx

```diff
@@ -0,0 +1,78 @@
+import _ from 'lodash';
+import React, {useCallback, useMemo} from 'react';
+import {View} from 'react-native';
+import type {ValueOf} from 'type-fest';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import type {ListItem} from '@components/SelectionList/types';
+import SelectionScreen from '@components/SelectionScreen';
+import type {SelectorType} from '@components/SelectionScreen';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Connections from '@libs/actions/connections';
+import Navigation from '@navigation/Navigation';
+import type {WithPolicyConnectionsProps} from '@pages/workspace/withPolicyConnections';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+type MenuListItem = ListItem & {
+    value: ValueOf<typeof CONST.XERO_CONFIG.INVOICE_STATUS>;
+};
+
+function XeroPurchaseBillStatusSelectorPage({policy}: WithPolicyConnectionsProps) {
+    const {translate} = useLocalize();
+    const policyID = policy?.id ?? '';
+    const styles = useThemeStyles();
+    const {billStatus} = policy?.connections?.xero?.config?.export ?? {};
+    const invoiceStatus = billStatus?.purchase;
+
+    const data: MenuListItem[] = Object.values(CONST.XERO_CONFIG.INVOICE_STATUS).map((status) => ({
+        value: status,
+        text: translate(`workspace.xero.invoiceStatus.values.${status}`),
+        keyForList: status,
+        isSelected: invoiceStatus === status,
```

Comment:
question, we don't need the check to be case insensitive?

Reply:
nope, it's checking against the const values